src/components/GameAbout/GameAbout.module.scss

```diff
@@ -1,32 +1,119 @@
+.container {
+  display: flex;
+  width: 75rem;
```

Comment:
```suggestion
  max-width: 75rem;
```

overflows at this breakpoint
![image](https://github.com/user-attachments/assets/151d4566-a4c7-46c9-97f9-436d9422acd4)

I think you meant to set max-width here